packages/kbn-check-mappings-update-cli/current_fields.json

```diff
@@ -1129,6 +1129,63 @@
     "taskType"
   ],
   "telemetry": [],
+  "test-actions-export-hidden": [
```

Comment:
It looks like we need to exclude the test fixtures from this list. Or are we intentionally including them now?

Reply:
I don't understand what's going on. I already removed those types manually, and then I also updated [this line](https://github.com/elastic/kibana/blob/main/src/platform/packages/private/kbn-repo-packages/modern/package.js#L194), so that plugins on those are considered test plugins, thus filtered out.

Removing them again, to see if CI adds them back.